ui/apps/platform/src/Components/FilteredWorkflowSelector/FilteredWorkflowSelector.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import {
+    Flex,
+    Icon,
+    MenuToggleElement,
+    MenuToggle,
+    Select,
+    SelectList,
+    SelectOption,
+} from '@patternfly/react-core';
+import { CubesIcon } from '@patternfly/react-icons';
+
+import { FilteredWorkflowState, SetFilteredWorkflowState } from './types';
+
+const width = '330px';
```

Comment:
As a warm up to future evaluation of accessibility, can you verify that width of `Select` element is okay if you zoom in? If not, is `em` or `rem` an alternative?

> Except for captions and images of text, text can be resized without assistive technology up to 200 percent without loss of content or functionality.

https://www.w3.org/TR/WCAG21/#resize-text

Reply:
It looks good to me. I kept the same pattern as we see in Vuln Management - Workload CVEs.